release-process.md

```diff
@@ -327,6 +327,7 @@ Next, update the rest of the Spark website. See how the previous releases are do
 * update `_layouts/global.html` if the new release is the latest one
 * update `documentation.md` to add link to the docs for the new release
 * add the new release to `js/downloads.js` (attention to the order of releases)
+* add the new release to `site/static/versions.json` (attention to the order of releases) [for `spark version drop down` of the `PySpark` docs]
```

Comment:
is this file in the spark repo or the spark-website repo?

Reply:
hmm why do we need this file... The name and version are always the same

Reply:
cc @HyukjinKwon

Reply:
> hmm why do we need this file... The name and version are always the same

- This file will be used in feature `spark version drop down` of the `PySpark doc site`, and it will be shared among all versions of the docs. When we add an item here, the data in the dropdown boxes of all versions can dynamically obtain the latest version data.
- In the implementation of version 3.5.1, in fact, it only requires `versio`n, but in order to display the `name` and actual `version` in the dropdown menu (`name` for display; `version` for `build click URL`), we have specially designed this design here.

https://github.com/apache/spark/blob/30055f7059b50a020ff0202c8454a119124c564c/python/docs/source/_templates/version-switcher.html#L71
https://github.com/apache/spark/blob/30055f7059b50a020ff0202c8454a119124c564c/python/docs/source/_templates/version-switcher.html#L63-L65
https://github.com/apache/spark/blob/30055f7059b50a020ff0202c8454a119124c564c/python/docs/source/_templates/version-switcher.html#L32
<img width="684" alt="image" src="https://github.com/apache/spark-website/assets/15246973/31ff4a26-bd0f-42ad-a103-6c1700381ec0">
<img width="586" alt="image" src="https://github.com/apache/spark-website/assets/15246973/8be3cff5-3204-41f7-83db-5e5b23de0e20">